src/api/TaskAssignments.js

```diff
@@ -116,7 +113,7 @@ class TaskAssignments extends Base {
         }
 
         const requestData = {
-            data: { resolution_state: resolutionState, message },
+            data: { resolution_state: taskStatus },
```

Comment:
so we read `status` but write `resolution_state`?

Reply:
updated, we'll only be using `status` now